tfhe/src/shortint/server_key/bivariate_pbs.rs

```diff
@@ -47,14 +30,11 @@ fn ciphertexts_can_be_packed_without_exceeding_space_or_noise(
 ) -> Result<(), CheckError> {
     let final_degree = (lhs.degree * factor) + rhs.degree;
 
-    let max_degree =
-        MaxDegree::from_msg_carry_modulus(server_key.message_modulus, server_key.carry_modulus);
```

Comment:
can you check if we have other places where we still do that ?

Reply:
Use ct MaxDegree as sk MaxDegree may be smaller (to keep carry propagation margin)

Reply:
Or keep previous code which was correct

Reply:
indeed with a comment